proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -120,17 +120,17 @@ extension Animal {
 }
 ```
 
-Contrary to the intent, `Animal` is still assumed as "having an associated type requirement", which happens to be the only thing restraining the existential.
+Unfortunately, `Animal` is still "assumed" to have an associated type requirement, which is solely responsible for restraining the existential.
 
-One more counterproductive aspect to the semantic inconsistency is how authors could be lead to avoid refining useful protocols in fear of their own protocol not qualifying for an existential type.
+The current semantic inconsistency also discourages authors from refining their existing protocols with other, useful ones in fear of losing existential qualification.
```

Comment:
`Did this part feel wordy to you?`

Yes, I thought that "One more counterproductive aspect to the semantic inconsistency is how" could be shortened to a simple sentence that just shows that it adds information (otherwise it is a bit hard for me to follow). Also, "could be lead" complicates the sentence due to the hypothetical speech.


Does the following make more sense?

----

The current semantic inconsistency also discourages authors from refining other, useful protocols with their existing ones in fear that they will lose existential qualification.